os-cherry-pop.py

```diff
@@ -236,20 +236,27 @@ def main():
     upstream_branches = list_branches_on_remote(upstream_remote)
     upstream_branches.sort()
 
-    release_index = upstream_branches.index("%s%s" % (STABLE_PREFIX, release))
+    release_index = -1
+    if release == "master":
+        release_index = upstream_branches.index(release)
+    else:
+        release_index = upstream_branches.index("%s%s" % (STABLE_PREFIX, release))
 
     if not previous_release:
         previous_upstream_branch = upstream_branches[release_index - 1]
         previous_release = previous_upstream_branch[len(STABLE_PREFIX):]
 
-    upstream_branch = "stable/{}".format(previous_release)
+    upstream_branch = upstream_branch = "stable/{}".format(previous_release)
+    if release == "master":
+        upstream_branch = "master"
```

Comment:
nit: This is a different pattern to [these lines](https://github.com/stackhpc/naming-things-is-hard/pull/16/files#diff-814e0faf2a27d00e35782e218e81100f6ecd3c1d752e6d898b0cf07f201b124eR240-R243) I don't mind which pattern you follow, but it would be nice if they were consistent